### ISSUE TITLE
Sticker pack folders

### ISSUE THREAD

**Issue by evinnichenko** [telegramdesktop/tdesktop#27022]
### Is your feature request related to a problem?

Often it is problematic to find a specific sticker when you have 20+ packs, with several of them having similar names. E.g. I just spent a good minute just trying to find a specific 👍 sticker from one of 4? 40k sticker packs that I have.

### Describe the solution you'd like

A way for users to organize sticker packs into folders, in a 3-stage hierarchy (folder-pack-sticker)(or 3+ stage: folder-subfolder-sub...subfolder-pack-sticker). This will allow users to organize stickers into thematic packs that are easier to navigate than the current single row at the bottom of sticker menu, ordered by recent.

### Describe alternatives you've considered

Decreasing number of packs: I like them all and don't want to miss out in case I want to use them sometime
Making my own pack of favorites: See above, not to mention it will become very bloated very quickly.

**Comment by LaysDragon** [comment]
yes,I really need this. I have so many sticker pack for different group,the stickers just too much to browsering when chat. Having folders to  organize sticker packs should solve this situation.

**Comment by carafelix** [comment]
bump

**Comment by ginqus** [comment]
bump. stickers have got to be the most irritating thing in telegram for me, folders would make the experience way better

**Comment by carafelix** [comment]
could be nice to also have folders of stickers which function just a pack of stickers but collected from other sticker packs. without making a new sticker pack of course. something like tagging stickers
